README.md

```diff
@@ -108,6 +108,8 @@ For several 'standard' use cases the [highfive/H5Easy.hpp](include/highfive/H5Ea
     - [Eigen::Matrix](http://eigen.tuxfamily.org) (optional, enable CMake option `HIGHFIVE_USE_EIGEN`),
     - [xt::xarray](https://github.com/QuantStack/xtensor) and [xt::xtensor](https://github.com/QuantStack/xtensor)
       (optional, enable CMake option `HIGHFIVE_USE_XTENSOR`).
+    - [cv::Mat_](https://docs.opencv.org/master/df/dfc/classcv_1_1Mat__.html)
+      (optional, enable CMake option `HIGHFIVE_USE_OPENCV`).
```

Comment:
It's borring that it is enable for H5Easy only. It should be in the core part.

Reply:
I'm all for integrating these third-party (i.e. opencv, Eigen, and xtensor) overloads to the core library. Historically I got the feeling that HighFive's development team favoured separating them in H5Easy. But if that has changed (or I misunderstood) I think it would be more than great. In that case I propose (some) separated PRs that I'm happy to work on